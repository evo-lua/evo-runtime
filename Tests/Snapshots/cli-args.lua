local arguments = { ... }

print("Number of command-line arguments received:", #arguments)

print("Dumping command-line arguments...")
dump(arguments)

print("Iterating over command-line arguments...")
for index, argument in pairs(arguments) do
	print(index, argument)
end