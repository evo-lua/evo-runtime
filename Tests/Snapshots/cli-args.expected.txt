Number of command-line arguments received:     4
Dumping command-line arguments...
{ "cli", "args", "go", "here" }
Iterating over command-line arguments...
1       cli
2       args
3       go
4       here